bundles/org.openhab.binding.mqtt.generic/xtend_examples.md

```diff
@@ -32,6 +32,22 @@ Thing mqtt:topic:WorkBroker:WorkSonoff "Work Sonoff" (mqtt:broker:WorkBroker) @
         Type switch : WorkLightTele "Work Tele" [ stateTopic="tele/worklight/STATE", transformationPattern="JSONPATH:$.POWER" ]
 }
 ```
+tasmota.things: Example of a Tasmota Device with Availablity-Topic state and standard Online/Offline message-payload
+```xtend
+Bridge mqtt:broker:mybroker [ host="mybroker", secure=false ]
```

Comment:
Would it be possible to change the "host" parameter value to "localhost" or, even better, to "192.168.0.42"? It is just to make more clear that the parameter should contain an IP address or a resolvable DNS name, and not the name that we have given to our Bridge (mqtt:broker:**mybroker**). Thanks Johanness!

Reply:
No Problem - sure...